pkg/volume/util/util.go

```diff
@@ -520,6 +520,10 @@ func GetPath(mounter volume.Mounter) (string, error) {
 // This means that a StatefulSet's volumes (`claimname-statefulsetname-id`) will spread across available zones,
 // assuming the id values are consecutive.
 func ChooseZoneForVolume(zones sets.String, pvcName string) string {
+	if zones.Len() == 0 {
```

Comment:
Thanks for finding and adding this! Can the same logic be also added to `ChooseZonesForVolume` below?

Reply:
Yep, of course

Reply:
Fixed in the new commits

Reply:
is this code change related to this PR?